### PR TITLE
Issue#287 cropper org creation

### PR DESCRIPTION
**Pull request by jenna1k**
related to #287 

implemented cropper in oraganization creation page

### CODE REVIEW

File: libs/organization/src/lib/components/organization-create/organization-create.component.scss [hunk 1]
```diff
@@ -44,3 +44,9 @@ button {
     width: 300px;
   }
 }
+
+drop-cropper {
+  display: block;
+  width: 100%;
+  height: 80%;
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/components/organization-create/organization-create.component.scss]: Hmm you shouldn't need to set the height && the width. I think you only want to set the width here.

File: libs/ui/src/lib/cropper/cropper/cropper.component.html [hunk 2]
```diff
@@ -7,6 +7,9 @@ <h3>Drop something here</h3>
     <image-cropper
       [imageFile]="file"
       [maintainAspectRatio]="true"
+      [containWithinAspectRatio]="true"
+      [onlyScaleDown]="true"
+      [alignImage]="'left'"
```
Comment (GrandSchtroumpf) [inline libs/ui/src/lib/cropper/cropper/cropper.component.html]: Remove that line

File: libs/ui/src/lib/cropper/cropper/cropper.component.html [hunk 3]
```diff
@@ -7,6 +7,9 @@ <h3>Drop something here</h3>
     <image-cropper
       [imageFile]="file"
       [maintainAspectRatio]="true"
```
Comment (GrandSchtroumpf) [inline libs/ui/src/lib/cropper/cropper/cropper.component.html]: You can just do : `<image-cropper ... maintainAspectRatio ...>`

File: libs/ui/src/lib/cropper/cropper/cropper.component.html (hunk 2 above)
Comment (jenna1k) [reply libs/ui/src/lib/cropper/cropper/cropper.component.html]: removed `[maintainAspectRatio]="true"` & `[alignImage]="'left'"`

**Pull request merged**